frontend/src/components/MapView/Layers/layer-utils.tsx

```diff
@@ -122,3 +124,28 @@ export const addPopupParams = (
     ),
   );
 };
+
+// Define the type for boundary data
+type BoundaryData = GeoJSON.FeatureCollection;
+
+// Define the function
```

Comment:
You have written more inspiring comments before :sweat_smile:

Reply:
🤖  haha